dasharo-security/tpm-support.robot

```diff
@@ -58,81 +130,44 @@ TPM001.003 TPM Support (Windows)
     Should Contain    ${tpm_ready}    True
     Should Contain    ${tpm_enabled}    True
 
-TPM002.001 Verify TPM version (firmware)
-    [Documentation]    This test aims to verify that the TPM version is
-    ...    correctly recognized by the firmware.
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM002.001 not supported
-    Prepare TPM Test On Ubuntu
-    ${result}=    Run Keyword And Ignore Error    Validate Expected TPM Chip Via Cbmem Console Log
-    IF    '${result}[0]' == 'FAIL'
-        Log To Console    \nChip detection failed, attempting cbmem log detection\n
-        Validate Expected TPM Version Via Cbmem TPM Eventlog
-    END
-
-TPM002.002 Verify TPM version (Ubuntu)
+TPM002.301 Verify TPM version (Windows)
     [Documentation]    This test aims to verify that the TPM version is
     ...    correctly recognized by the operating system.
-    [Tags]    minimal-regression
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM002.002 not supported
-    Prepare TPM Test On Ubuntu
-    Validate Expected TPM Version Via Sysfs
-
-TPM002.003 Verify TPM version (Windows)
-    [Documentation]    This test aims to verify that the TPM version is
-    ...    correctly recognized by the operating system.
-    Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    TPM002.003 not supported
+    ...    Previous IDs: TPM002.003
+    Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    TPM002.301 not supported
     Power On
     Login To Windows
     ${out}=    Execute Command In Terminal
     ...    wmic /namespace:\\\\root\\cimv2\\security\\microsofttpm path win32_tpm get * /format:textvaluelist.xsl
     Should Contain    ${out}    SpecVersion=2.0
 
-TPM003.001 Check TPM Physical Presence Interface (firmware)
-    [Documentation]    This test aims to verify that the TPM Physical Presence
-    ...    Interface is supported by the firmware and the log can be detected
-    ...    with cbmem within Ubuntu
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM003.001 not supported
-    Prepare TPM Test On Ubuntu
-    ${out}=    Execute Command In Terminal    cbmem -1 | grep PPI
-    Should Contain    ${out}    PPI: Pending OS request
-    Should Contain    ${out}    PPI: OS response
-
-TPM003.002 Check TPM Physical Presence Interface (Ubuntu)
-    [Documentation]    This test aims to verify that the TPM Physical Presence
-    ...    Interface is correctly recognized by the operating system.
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM003.002 not supported
-    Prepare TPM Test On Ubuntu
-    ${out}=    Execute Command In Terminal    cat /sys/class/tpm/tpm0/ppi/version
-    IF    '${TPM_SUPPORTED_VERSION}' == '1'
-        Should Contain    ${out}    1.2
-    ELSE IF    '${TPM_SUPPORTED_VERSION}' == '2'
-        Should Contain    ${out}    1.3
-    ELSE
-        Fail    Invalid expected version, please verify config
-    END
-
 TPM003.003 Check TPM Physical Presence Interface (Windows)
```

Comment:
This wasn't changed. Jakub did it in https://github.com/Dasharo/open-source-firmware-validation/pull/732/commits/b3373269c35bc0323a40c78ef0cf65b630f2d452, just leaving the comment in case someone else notices it.